packages/coin-support-xrp/src/operations/createAccounts/schemes/types.ts

```diff
@@ -0,0 +1,12 @@
+import { IDerivationScheme } from '@cypherock/coin-support-interfaces';
+
+export const XrpDerivationSchemeMap = {
+  default: '',
```

Comment:
Update `packages/cysync-core/src/dialogs/EditAccountDialog/Dialogs/AccountDetails.tsx` to not only show `derivationScheme` when it has a non empty string.